libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts

```diff
@@ -242,6 +188,14 @@ export const aboutTheAccidentSection = buildSection({
                 },
               ],
             }),
+            buildAlertMessageField({
+              id: 'attachments.injuryCertificate.alert',
+              title: application.general.warningTitle,
```

Comment:
Shouldn't this text be specific to this use case not a general title and message?

Reply:
I'll move from general to labels